bundles/org.eclipse.equinox.jsp.jasper.registry/META-INF/MANIFEST.MF

```diff
@@ -9,9 +9,7 @@ Bundle-Activator: org.eclipse.equinox.internal.jsp.jasper.registry.Activator
 Import-Package: org.eclipse.equinox.jsp.jasper,
  org.osgi.framework;version="1.3.0",
  org.osgi.service.packageadmin;version="1.2.0",
- org.osgi.util.tracker;version="[1.5.0,2)",
- javax.servlet;version="2.4",
- javax.servlet.http;version="2.4"
```

Comment:
This currently triggers an indirectly referenced error during compilation.
Neither bnd detects this in the classfile nor PDE discover it from source, need to investigate how to handle this issue.

Reply:
There is no error in Jenkins anymore. Should it be merged now?